livedata/src/main/java/io/getstream/chat/android/livedata/utils/DefaultRetryPolicy.kt

```diff
@@ -9,7 +9,7 @@ class DefaultRetryPolicy : RetryPolicy {
         attempt: Int,
         error: ChatError
     ): Boolean {
-        return attempt < 5
+        return attempt < 3
```

Comment:
Could we extract this magic number?

Reply:
nothing magic about it, just the default is 3 retries.